tests/WebAppCallsMicrosoftGraph/MicrosoftGraphServiceExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Graph;
+using Microsoft.Identity.Web;
+
+namespace WebAppCallsMicrosoftGraph
+{
+    public static class MicrosoftGraphServiceExtensions
+    {
+        public static void AddMicrosoftGraph(this IServiceCollection services, IConfiguration configuration)
+        {
+            services.AddSingleton<GraphServiceClient, GraphServiceClient>(serviceProvider =>
```

Comment:
`IGraphServiceClient`?
```suggestion
            services.AddSingleton<IGraphServiceClient, GraphServiceClient>(serviceProvider =>
```

Reply:
@pmaytak. The Graph SDK team does not want customer to use the interface but directly the GraphServiceClient. @darrelmiller can you please explain why?

Reply:
Hmm. Interesting.

Reply:
IGraphServiceClient contains an entry point for each top level entityset or singleton.  Adding a new top level on the resource is not considered a breaking change and can happen fairly regularly.  Changing an interface is a breaking change.  This would be very undesirable behavior for a customer to update to a new version of the Graph SDK and find their implementations of IGraphServiceClient keep breaking.

Additionally, it is not a necessary interface.  There is no reason for creating an alternate implementation of GraphServiceClient other than for testing and there are better ways of doing testing than attempting to mock that interface.

Reply:
thanks for the explanation @darrelmiller

Reply:
reverted to use GraphServiceClient per @darrelmiller's guidance